lib/src/arg_parser_exception.dart

```diff
@@ -9,6 +9,10 @@ class ArgParserException extends FormatException {
   /// This will be empty if the error was on the root parser.
   final List<String> commands;
 
-  ArgParserException(super.message, [Iterable<String>? commands])
+  /// The argument that were being parsed when the error was discovered.
```

Comment:
[nit] Is this the argument _name_ or argument _value_?

```suggestion
  /// The argument name that was being parsed when the error was discovered.
```

Reply:
Thank you for the review! This is the argument value.

Reply:
> This is the argument value.

In that case we should change this comment, my edit was assuming it was the name, not the value.

Reply:
> In that case we should change this comment, my edit was assuming it was the name, not the value.

Thank you for the clarification. I referred to it as the argument value, but I actually meant the name of the argument value (e.g., `--path` or `--input`), so the change is correct.

Also, I apologize if my explanation wasn't clear—English is not my first language.